tests/qa_automation/qa_run.pm

```diff
@@ -0,0 +1,100 @@
+package qa_run;
```

Comment:
Well yeah, that fixes it, but how about using i. e. `lib/automation` and `lib/x11`. And then in tests call `use base "automation/qa_run"`?